src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/New-AzureRmRecoveryServicesAsrReplicationProtectedItem.md

```diff
@@ -47,6 +47,15 @@ New-AzureRmRecoveryServicesAsrReplicationProtectedItem [-HyperVToAzure] -Protect
  [-Confirm] [<CommonParameters>]
 ```
 
+### AzureToAzure
```

Comment:
Please add an example for this new parameter set

Reply:
sure.